src/components/views/QuizQuestion.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import Chord from '../Chord'
+import Choice from '../Choice'
+import {
+  Container,
+  Row,
+  Col,
+} from 'shards-react'
+import useResponsiveStyles from '../../hooks/useResponsiveStyles'
+
+
+
+export default function QuizQuestion(props) {
+  const sizedStyles = useResponsiveStyles()
+  const { borderRadius, questionText } = sizedStyles
+  const { chord, question, colors, handleClick, onKeyPressed, currentInput } = props
+  const noteColors = (() => {
```

Comment:
Would this be any more concise as:

⚠️ Untested js code 😬 

```Javascript
const noteColor = (() => {
    return colors
        .filter(input => input.color === 'green')
        .map(input => input.input)
)()
```

If this is valid (which I am not sure if it is!), it seems like we could get rid of a logical branch, here.

Reply:
(Also, ignore any unidiomatic formatting…)

Reply:
I think that works.  If the syntax isn't valid, certainly something with that logic is. 

Fwiw, I threw this in quickly as a placeholder when I refactored `Quiz.js`. I'm not sure it'll stay. Originally there were 3 state variables, `noteColors, green, red`, and I realized (a) that `noteColors` and `green` were the same array and (b) that the state update logic back in `Quiz.js` was a mess. Basically, you have to update a state variable (or a couple other things) to trigger a re-render, which I needed after every input to show the red or green button color. I realized it made a lot more sense to have a single colors object that updates on each click, but I haven't yet finished the ripple effects of that refactor down the tree.